Util/function/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 
 dependencies {
     api project(':qst-type')
+    api project(':engine-query-constants')
```

Comment:
We aren't re-exposing the constants, or public methods from engine-query-constants; this could be an implementation dependency.